test/utils/Defaults.sol

```diff
@@ -25,10 +25,12 @@ contract Defaults is Constants {
     uint40 public immutable END_TIME;
     uint256 public constant MAX_SEGMENT_COUNT = 300;
     uint40 public immutable MAX_SEGMENT_DURATION;
+    uint256 public constant MAX_TRANCHE_COUNT = 300;
```

Comment:
Should we use a common variable to represent both `MAX_TRANCHE_COUNT` and `MAX_TRANCHE_COUNT`? Since their values are same, it is leading code duplicity. 

How about `MAX_SECTIONS`?

Reply:
yeah, we can, what do you say about `MAX_COUNT`?

Reply:
Sounds good.